Formula/subversion.rb

```diff
@@ -124,6 +124,12 @@ def install
       Formula["zlib"].opt_prefix
     end
 
+    perl = if OS.mac?
+      "/usr/bin/perl"
+    else
+      "#{Formula["perl"].opt_bin}/perl"
+    end
```

Comment:
```suggestion
    perl = DevelopmentTools.locate("perl")
```
We can probably do the same with `ruby` below.

Reply:
This is updated, too-- both `perl` and `ruby`.